Orange/regression/tree.py

```diff
@@ -1,12 +1,142 @@
+"""Tree inducers: SKL and Orange's own inducer"""
+import numpy as np
 import sklearn.tree as skl_tree
 
-from Orange.base import Tree
-from Orange.regression import SklLearner, SklModel
+from Orange.tree import Node, DiscreteNode, MappedDiscreteNode, \
+    NumericNode, OrangeTreeModel
+from Orange.regression import SklLearner, SklModel, Learner
+from Orange.classification import _tree_scorers
 
-__all__ = ["TreeRegressionLearner"]
+__all__ = ["TreeRegressionLearner", "OrangeTreeLearner"]
 
 
-class TreeRegressor(SklModel, Tree):
+class OrangeTreeLearner(Learner):
```

Comment:
... yes, but only when you need to import both in the same file. I would prefer, though, to merge both classes into a single class for classification and regression. Same for widget.